packages/@lwc/compiler/src/transformers/__tests__/transform-javascript.spec.ts

```diff
@@ -269,3 +269,34 @@ describe('sourcemaps', () => {
         });
     });
 });
+
+describe('transformFile function', () => {
+    it('should be tranform Typescript file with ext (.tsx)', () => {
+        const src = `
+        import { LightningElement } from 'lwc';
+        export default class Foo extends LightningElement {}
+    `;
```

Comment:
```suggestion
        const src = `
            import { LightningElement } from 'lwc';
            export default class Foo extends LightningElement {}
        `;
```

Just slightly nicer formatting.